src/ui/buttons/styles/responsive.js

```diff
@@ -132,6 +136,25 @@ export function buttonResponsiveStyle({ height } : {| height? : ?number |}) : st
                 }
             }
 
+            @media only screen and (min-width: ${ style.minWidth }px) and (max-width: 320px) {
+
+                .${ CLASS.CONTAINER } .${ CLASS.BUTTON_ROW } .${ CLASS.TEXT }, .${ CLASS.CONTAINER } .${ CLASS.BUTTON_ROW } .${ CLASS.SPACE } {
```

Comment:
Do we need the class container included as well? Wondering if we can shorten up this selector like so:
```suggestion
                .${ CLASS.BUTTON_ROW } .${ CLASS.TEXT }, .${ CLASS.BUTTON_ROW } .${ CLASS.SPACE } {
```

Reply:
Actually If the container class is removed the one in line 38 will be used

Reply:
ah good point.